### ISSUE TITLE
Do you have a model of space bar and iso return?

### ISSUE THREAD

**Issue by yorkc** [mjseabright/keycaps#1]

**Comment by mjseabright** [comment]
Not currently. I own an SA spacebar so I can add a model of that if it would be useful to you. Unfortunately I don't own any iso returns or a DSA spacebar so I can't measure them for modelling.

If you want an SA spacebar, just say and I'll try to add one.

**Comment by yorkc** [comment]
Yes, I do. I need the 6.25u and 7u space bar. Thank u very much~

**Comment by mjseabright** [comment]
Ok, I'll try to get that done over the weekend. Will update here when it's done.

Out of curiosity, what are you using the models for?

**Comment by mjseabright** [comment]
SA spacebars of both 6.25u and 7u have been added.

**Comment by yorkc** [comment]
I'll use these models for rendering.U can check here: [https://geekhack.org/index.php?topic=95597.0](https://geekhack.org/index.php?topic=95597.0)
Thank you for your help~

**Comment by mjseabright** [comment]
Awesome, that's a really nice set. Great work. I'm glad I could help.

**Comment by yorkc** [comment]
Thank you for your comments, lol

**Comment by mjseabright** [comment]
I've just added a stepped caps lock key which may be useful to you.

**Comment by yorkc** [comment]
Ok, thank you very much again~
